tests/unhappypath/unhappypath_test.go

```diff
@@ -0,0 +1,309 @@
+package tests
+
+import (
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	storkv1 "github.com/libopenstorage/stork/pkg/apis/stork/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	"github.com/onsi/ginkgo/reporters"
+	. "github.com/onsi/gomega"
+	"github.com/portworx/sched-ops/k8s/stork"
+	"github.com/portworx/torpedo/drivers/node"
+	"github.com/portworx/torpedo/drivers/scheduler"
+	"github.com/portworx/torpedo/drivers/volume"
+	"github.com/portworx/torpedo/pkg/testrailuttils"
+	. "github.com/portworx/torpedo/tests"
+	"github.com/sirupsen/logrus"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	storkops = stork.Instance()
+)
+
+const (
+	dropPercentage      = 20
+	delayInMilliseconds = 250
+	//24 hours
+	totalTimeInHours = 24
+	// TODO need to make it 60 minutes
+	errorPersistTimeInMinutes     = 10 * time.Minute
+	snapshotScheduleRetryInterval = 10 * time.Second
+	snapshotScheduleRetryTimeout  = 3 * time.Minute
+)
+
+func TestBasic(t *testing.T) {
+	RegisterFailHandler(Fail)
+
+	var specReporters []Reporter
+	junitReporter := reporters.NewJUnitReporter("/testresults/junit_basic.xml")
+	specReporters = append(specReporters, junitReporter)
+	RunSpecsWithDefaultAndCustomReporters(t, "Torpedo : Basic", specReporters)
+}
+
+var _ = BeforeSuite(func() {
+	InitInstance()
+	// package installation hangs when using nsenter to run command on nodes
+	if Inst().N.IsUsingSSH() {
```

Comment:
These steps should be part of VM installation or creation. Better to put them in VM template or part of spawn.

Reply:
Opened ticket https://portworx.atlassian.net/browse/PTX-12174 to have this utility on the template iteself.